index.html

```diff
@@ -1588,8 +1613,9 @@
               </ol>
             </li>
             <li>If the user agent is unable to continuously <a>monitor the list
-            of available presentation displays</a> but can find presentation
-            displays in order to start a connection, then:
+            of available presentation displays</a> at the current time but can
```

Comment:
I wonder whether we could further clarify the two "unable to monitor" steps:

1. I would switch them in the algorithm so that the step that rejects the promise appears first: it would make it clearer that a controlling browsing context that does not support background monitoring does not need to evaluate the second clause and will never resolve the promise with a newly created `PresentationAvailability` object in particular.
2. I'm not sure I understand "at the current time". Isn't "in the background" clearer, consistent with section 6.4, and what is meant here?